src/test/java/com/connexta/ingest/controllers/IngestControllerTests.java

```diff
@@ -110,7 +118,18 @@ public void testMissingCorrelationID() throws Exception {
   @Test
   public void testMissingAcceptVersionHeader() throws Exception {
     mockMvc
-        .perform(multipart("/ingest").file(FILE).file(METACARD))
+        .perform(
+            multipart("/ingest").file(FILE).file(METACARD).header("Last-Modified", LAST_MODIFIED))
+        .andExpect(status().isBadRequest());
+
+    verifyNoInteractions(mockIngestService);
+  }
+
+  @Test
+  public void testMissingLastModifiedHeader() throws Exception {
```

Comment:
✏️ Add a (parameterized) test to verify that 400 is returned when the format of the `Last-Modified` header is invalid.